pyproject.toml

```diff
@@ -49,7 +49,7 @@ alembic = "~1.12.1"
 Babel = "~2.13.1"
 fastapi = "~0.103.1"
 jinja2 = "~3.1.3"
-openfoodfacts = "~0.2.0"
```

Comment:
We will release a new version of Open Food Facts python SDK to have a clean version to pin